queries.sql

```diff
@@ -15,3 +15,50 @@ select * from animals where neutered = true
 select * from animals where name not like '%Gabumon'
 
 select * from animals where weight_kg between  10.4 and 17.3
+
```

Comment:
Great job @r-ahmed2022 🤝

- [ ] Kindly add all queries for this milestone to the `queries.sql` file.

Reply:
done, as advised